notebooks/110-ct-segmentation-quantize/110-ct-segmentation-quantize-nncf.ipynb

```diff
@@ -453,16 +447,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 21,
    "metadata": {
     "collapsed": false
    },
    "outputs": [],
    "source": [
     "fp32_ir_path = MODEL_DIR / Path('unet_kits19_fp32.xml')\n",
     "\n",
-    "fp32_ir_model = mo.convert_model(model, input_shape=(1, 1, 512, 512))\n",
-    "serialize(fp32_ir_model, str(fp32_ir_path))"
+    "fp32_ir_model = ov.convert_model(model, example_input=torch.ones(1, 1, 512, 512, dtype=torch.float32))\n",
```

Comment:
`float32` is the default tensor data type.

Reply:
Anyway, it is explicit in this form